### ISSUE TITLE
Old version of react-navigation library WARNING

### ISSUE THREAD

**Issue by butri71** [react-navigation/react-navigation#9252]
I've been running my App on Expo using react-navigation V5.

Since i upgraded expo to SDK 40 I started receiving the following warning saying that all my react-navigation libraries are old.

Message:
_It appears that you are using old version of react-navigation library. Please update @react-navigation/bottom-tabs, @react-navigation/stack and @react-navigation/drawer to version 5.10.0 or above to take full advantage of new functionality added to react-native-screens_

My version of those libraries is definitely older than the ones suggested by the warning message:

"@react-navigation/native": "^5.8.10",
"@react-navigation/bottom-tabs": "^5.11.2",
"@react-navigation/stack": "^5.12.8",
"@react-navigation/drawer": "^5.11.4",

This annoying message is appearing all the time in the console.

**Comment by butri71** [comment]
I've just installed the latest version directly from NPM and in my package I still see the following versions to be the latest available which btw are all above 5.10

"@react-navigation/native": "^5.8.10",
"@react-navigation/bottom-tabs": "^5.11.2",
"@react-navigation/stack": "^5.12.8",
"@react-navigation/drawer": "^5.11.4",

**Comment by wicharek** [comment]
I am getting the same warning in react-navigation 4.x. Honestly, don't get the purpose of issuing this warning.

Documentation states:

```
If React Navigation 4 is working well for you and you don't need any of the new capabilities of the new version, you can keep using it. You don't have to rewrite your navigation structure.
```

And then the library throwing the warning into the developer's face at every refresh.

**Comment by omaksiuta** [comment]
Still issue
[22:56:30] It appears that you are using old version of react-navigation library. Please update @react-navigation/bottom-tabs, @react-navigation/stack and @react-navigation/drawer to version 5.10.0 or above to take full advantage of new functionality added to react-native-screens
My Config

{
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "eject": "expo eject",
    "server": "npm run start --prefix server"
  },
  "dependencies": {
    "@expo/vector-icons": "^12.0.0",
    "@kalwani/react-native-modal": "^0.1.2",
    "@react-native-community/datetimepicker": "3.2.0",
    "@react-native-community/masked-view": "^0.1.10",
    "@react-native-community/push-notification-ios": "^1.0.2",
    "@react-navigation/bottom-tabs": "^5.11.11",
    "@types/react-native-snap-carousel": "^3.8.1",
    "audit": "0.0.6",
    "axios": "^0.21.1",
    "babel-plugin-module-resolver": "^3.2.0",
    "engine.io-client": "^4.0.5",
    "expo": "^41.0.0",
    "expo-av": "~9.1.2",
    "expo-camera": "~11.0.2",
    "expo-constants": "~10.1.3",
    "expo-device": "~3.2.0",
    "expo-document-picker": "~9.1.2",
    "expo-facebook": "~11.0.5",
    "expo-file-system": "~11.0.2",
    "expo-font": "~9.1.0",
    "expo-image-manipulator": "~9.1.0",
    "expo-image-picker": "~10.1.4",
    "expo-localization": "~10.1.0",
    "expo-location": "~12.0.4",
    "expo-network": "~3.1.1",
    "expo-permissions": "~12.0.1",
    "expo-screen-capture": "^2.0.0",
    "expo-status-bar": "~1.0.4",
    "fix": "0.0.6",
    "i18n-js": "^3.5.1",
    "jwt-decode": "^2.2.0",
    "lodash": "^4.17.19",
    "metro-core": "^0.60.0",
    "moment": "^2.24.0",
    "moment-timezone": "^0.5.31",
    "native-base": "2.13.14",
    "paths-js": "^0.4.10",
    "react": "16.13.1",
    "react-dom": "16.13.1",
    "react-native": "0.63.4",
    "react-native-appearance": "~0.3.3",
    "react-native-auto-height-image": "^3.1.0",
    "react-native-calendar-strip": "^2.0.9",
    "react-native-calendars": "^1.1264.0",
    "react-native-chart-kit": "^5.1.1",
    "react-native-countdown-component": "^2.7.1",
    "react-native-drawer": "^2.5.1",
    "react-native-elements": "^2.0.2",
    "react-native-expo-cached-image": "^1.3.1",
    "react-native-flash-message": "^0.1.15",
    "react-native-gesture-handler": "~1.10.2",
    "react-native-gifted-chat": "^0.16.1",
    "react-native-indicators": "^0.17.0",
    "react-native-maps": "0.27.1",
    "react-native-maps-directions": "^1.7.6",
    "react-native-masked-text": "^1.13.0",
    "react-native-modal-datetime-picker": "^8.7.0",
    "react-native-numeric-input": "^1.8.3",
    "react-native-pathjs-charts": "0.0.34",
    "react-native-popup-menu": "^0.15.9",
    "react-native-progress-bar-animated": "^1.0.6",
    "react-native-reanimated": "~2.1.0",
    "react-native-safe-area-context": "^3.2.0",
    "react-native-scalable-image": "^1.0.0",
    "react-native-screens": "~3.0.0",
    "react-native-scroll-bottom-sheet": "^0.6.3",
    "react-native-scrollview-offset": "^1.0.0",
    "react-native-shape": "^1.1.2",
    "react-native-size-matters": "^0.3.0",
    "react-native-snap-carousel": "^3.9.0",
    "react-native-star-rating": "^1.1.0",
    "react-native-svg": "12.1.0",
    "react-native-svg-charts": "^5.3.0",
    "react-native-text-input-mask": "^2.0.0",
    "react-native-unimodules": "~0.13.3",
    "react-native-vector-icons": "^6.6.0",
    "react-native-web": "^0.17.1",
    "react-navigation": "^3.13.0",
    "react-navigation-stack": "^2.10.4",
    "react-navigation-tabs": "^2.11.1",
    "socket.io-client": "^2.3.0",
    "vue-native-core": "0.2.0",
    "vue-native-helper": "0.1.4",
    "vuelidate": "^0.7.5",
    "vuex": "^3.1.2"
  },
  "devDependencies": {
    "@babel/core": "^7.14.6",
    "babel-preset-expo": "8.3.0",
    "vue-native-scripts": "0.1.4"
  },
  "private": true
}

**Comment by DianaKoenraadt** [comment]
This is the most annoying warning I've ever encountered. It floods my console and makes it impossible to debug anything else.
